src/Service/DynamoDb/src/DynamoDbClient.php

```diff
@@ -35,6 +37,28 @@
 
 class DynamoDbClient extends AbstractApi
 {
+    /**
+     * The `BatchWriteItem` operation puts or deletes multiple items in one or more tables. A single call to
+     * `BatchWriteItem` can write up to 16 MB of data, which can comprise as many as 25 put or delete requests. Individual
+     * items to be written can be as large as 400 KB.
+     *
+     * @see https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-dynamodb-2012-08-10.html#batchwriteitem
+     *
+     * @param array{
+     *   RequestItems: array<string, array>,
```

Comment:
This is not super pretty. But Im okey with this.

Reply:
should be fixed by https://github.com/async-aws/aws/pull/681